### PR TITLE
Support multiple brews, casks or taps in DSL

### PR DESCRIPTION
**Pull request by paulfri**
Would like to simplify my Brewfile a bit.

### CODE REVIEW

File: lib/bundle/dsl.rb [hunk 1]
```diff
@@ -57,16 +57,28 @@ def install
...
     end
 
-    def brew(name, options={})
-      @entries << Entry.new(:brew, name, options)
+    def brew(*args)
+      names, options = if args.last.is_a? Hash
+        [args[0...-1], args.last]
+      else
+        [args, {}]
+      end
```
Comment (xu-cheng) [inline lib/bundle/dsl.rb]: Maybe just
```ruby
options = args.last.is_a?(Hash) ? args.pop : {}
```

File: spec/dsl_spec.rb [hunk 2]
```diff
@@ -7,4 +7,90 @@
...
+        Bundle::Dsl.new("tap 'phinze/cask', 'homebrew/versions'")
+      end
+    end
+
+    context 'when array of taps is passed' do
+      it 'adds entries for each tap' do
+        Bundle::Dsl.new('tap %w(phinze/cask homebrew/versions)')
+      end
+    end
+  end
```
Comment (xu-cheng) [inline spec/dsl_spec.rb]: :+1: for test.

**Comment by xu-cheng** [comment]
LGTM. Will let other maintainers have a look.

**Comment by mikemcquaid** [comment]
> Would like to simplify my Brewfile a bit.

Can you give an example of what it looks like looks like before and after this change?

**Comment by andrew** [comment]
Would be good to add some documentation to the readme as well.

**Comment by paulfri** [comment]
I added a simple example to the readme. Let me know what you think.

File: Readme.md [hunk 3]
```diff
@@ -28,14 +28,15 @@ Create a `Brewfile` in the root of your project:
...
-    tap 'caskroom/cask'
+    tap 'caskroom/cask', 'homebrew/dupes'
     brew 'emacs', args: ['cocoa', 'srgb', 'with-gnutls']
     brew 'redis'
     brew 'mongodb'
     brew 'sphinx'
     brew 'imagemagick'
     brew 'mysql'
-    cask 'google-chrome'
+    brew 'tcl-tk', 'python', 'python3', args: ['universal', 'with-brewed-tk']
```
Comment (mikemcquaid) [inline Readme.md]: Do these all get installed with the same args?

**Comment by mikemcquaid** [comment]
I'm :-1: on this change, I think. I think it makes the DSL more confusing and ambigious and may limit us if we want to expand the DSL in future.

File: Readme.md (hunk 3 above)
Comment (paulfri) [inline Readme.md]: Yes, that's the idea. Homebrew ignores unapplicable arguments so it should be safe in e.g. this case.

Comment (mikemcquaid) [inline Readme.md]: Homebrew may not always do this; I don't think we should rely on this.

**Comment by paulfri** [comment]
@mikemcquaid thanks for your feedback!

If there's any suggestion for where you might want the DSL to go, I'll happily edit this PR, otherwise I'll deal with the duplication for the time being.

**Comment by mikemcquaid** [comment]
@paulfri Thanks for understanding :heart:

**Pull request closed without merging**